plugins/spi/SPIOutput.h

```diff
@@ -38,6 +38,22 @@ namespace spi {
 
 class SPIOutput: public ola::rdm::DiscoverableRDMControllerInterface {
  public:
+  // definitions for all SPI Personalities
+  // this hardcoded mapping is needed for backwards compatibility
+  // enum class SPI_PERSONALITY : unsigned int {
+  enum SPI_PERSONALITY {
+    PERS_WS2801_INDIVIDUAL = 1,
+    PERS_WS2801_COMBINED = 2,
+    PERS_LDP8806_INDIVIDUAL = 3,
+    PERS_LDP8806_COMBINED = 4,
+    PERS_P9813_INDIVIDUAL = 5,
+    PERS_P9813_COMBINED = 6,
+    PERS_APA102_INDIVIDUAL = 7,
+    PERS_APA102_COMBINED = 8,
+    PERS_APA102PB_INDIVIDUAL = 9,
```

Comment:
These two don't need it anyway, due to no backwards compatibility.

Also I wonder if PERS_APA102_PB_INDIVIDUAL would be clearer, as the LED type is still the same.